snippets/bridge_transfer.py

```diff
@@ -0,0 +1,100 @@
+"""CLI command for setting up a bridge."""
+
+from time import sleep
+
+from xrpl.account import does_account_exist, get_balance
+from xrpl.clients import JsonRpcClient
+from xrpl.models import (
+    AccountObjects,
+    AccountObjectType,
+    XChainAccountCreateCommit,
+    XChainBridge,
+    XChainCommit,
+    XChainCreateClaimID,
+)
+from xrpl.transaction import submit_and_wait
+from xrpl.utils import xrp_to_drops
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+locking_client = JsonRpcClient("https://sidechain-net1.devnet.rippletest.net:51234")
+issuing_client = JsonRpcClient("https://sidechain-net2.devnet.rippletest.net:51234")
+
+locking_chain_door = "rMAXACCrp3Y8PpswXcg3bKggHX76V3F8M4"
+bridge_data = locking_client.request(
+    AccountObjects(account=locking_chain_door, type=AccountObjectType.BRIDGE)
+).result["account_objects"][0]
+bridge = XChainBridge.from_xrpl(bridge_data["XChainBridge"])
+print(bridge)
+
+wallet1 = generate_faucet_wallet(locking_client, debug=True)
+wallet2 = Wallet.create()
+
+print(f"Creating {wallet2.classic_address} on the issuing chain via the bridge...")
```

Comment:
Similar comments to the xrpl.js PR apply here:
* Printing Wallet 1 being funded on the locking chain is helpful. 
* Printing before submitting transactions to indicate what's about to happen is helpful so snippet runners don't think their app just froze for no reason
* Same helper method to extract the `xchain_claim_id` seems helpful
* Likely also helpful to add an issue to add `signature_reward` to autofill